src/popup/components/Overview.vue

```diff
@@ -39,70 +39,29 @@
     </div>
 
     <div class="details">
```

Comment:
By the way of doing changes in this component I suggest to rename it. The name should not use only one word as this is not allowed in Vue 3. Also the name should be more precise and describe the purpose of this component.